lib/cli/context/type/module.rb

```diff
@@ -22,10 +22,18 @@ class Module < Context
         include Command::Module
         include Command::Service        
 
+        COMMAND_DEFS = [:service, :module]
+
         def add_command_defs!
-          add_command :module
-          # add_command :service needed for stage and deploy commands
-          add_command :service
+         COMMAND_DEFS.each {|cmd| add_command(cmd)}
+        end
+
+        def context_type
+          'module'
+        end
+
+        def allowed_commands_defs
+          ['service', 'module']
```

Comment:
Point of using COMMAND_DEFS constant is to reuse it in multiple places. So when returning ['service', 'module'] from allowed_commands_defs we are duplicating allowed commands. Instead we should use:
COMMAND_DEFS.map { |cmd| cmd.to_s }